build.gradle

```diff
@@ -22,95 +26,17 @@ ext {
 ext {
     // libs
     automatticAboutVersion = '1.4.0'
```

Comment:
1. Blocker (🚫): This and all versions should be migrated as well. Otherwise, the overall `Version Catalog` solution would end-up being half-baked into `trunk`.
2. Question (❓): Was there any problem that made this and the rest of the versions, along with their respective dependencies difficult for you to migrate? 🤔

Reply:
Yes @ParaskP7, I was confused in handling `implementation "$gradle.ext.aboutAutomatticBinaryPath:$automatticAboutVersion"`

I am not sure `gradle.ext.aboutAutomatticBinaryPath` what's this and where its taking from that bimary path. Can you help me over this?